content/docs/user-guide/assets/asset-processor/debugging.md

```diff
@@ -97,6 +347,7 @@ Use this tool to automatically attach the debugger to spawned child processes.
 1. In Visual Studio, start `AssetProcessor.exe`. Breakpoints in Asset Builders work as normal.
 
 ## Debug Asset Builders from Asset Processor 
+<a name="DebugAssetBuilders"></a>
```

Comment:
```suggestion
<a name="DebugAssetBuilders"/>
```

Reply:
Netlify-style anchor links work better. Using HTML links will cause the page to scroll too far.